common/src/main/kotlin/io/homeassistant/companion/android/common/data/integration/impl/IntegrationRepositoryImpl.kt

```diff
@@ -797,6 +668,42 @@ class IntegrationRepositoryImpl @AssistedInject constructor(
         }
     }
 
+    private suspend fun ServerConnectionInfo.callWebhookOnUrls(
```

Comment:
Why did you make this an extension function on `ServerConnectionInfo`? I don't see a reason to tightly couple it instead of a normal function. You only save a tiny bit of code in line 677.

Reply:
I guess my point was: `private suspend fun callWebhookOnUrls` also works and doesn't  add a lot of additional code (where you added `connection` in your latest commit, it's be `server.connection`)?

If it's private like this using extensions adds little value while it does create room for confusion IMO (where did the `connectiin` come from, if you don't look carefully at the function), but not a blocker.